datasets/Lab41-SRI-VOiCES.yaml

```diff
@@ -0,0 +1,29 @@
+Name: Voices Obscured in Complex Enrivonmental Settings (VOiCES)
+Description: |
+  VOiCES is a speech corpus recorded in acoustically challenging settings,
+  using distant microphone recording. Speech was recorded in real rooms with various
+  acoustic features (reverb, echo, HVAC systems, outside noise, etc.). Adversarial noise,
+  either television, music, or babble, was concurrently played with clean speech.
+  Data was recorded using multiple microphones strategically placed
+  throughout the room. The corpus includes audio recordings, orthographic transcriptions,
+  and speaker labels.
+Contact: https://github.com/voices18/utilities/issues
+UpdateFrequency: Data from two additional rooms will be added to the corpus Fall 2018.
+Tags: 
+  - aws-pds
+  - machine learning
+  - automatic speech recognition
+  - speaker identification
+  - denoising
+  - speech processing
+License: Creative Commons BY 4.0 (see [here](htpps://voices.lab41.org) for more details)
+Resources:
+  - Description: wav audio files, orthographic transcriptions, and speaker ID
+    ARN: arn:aws:::lab41openaudiocorpus
+    Region: us-east-1
+    Type: S3 Bucket
+DataAtWork:
+  - Description: Getting started with VOiCES data 
```

Comment:
Please change `Description` to `Title`

https://travis-ci.org/awslabs/open-data-registry/builds/392915947#L495